Components/User/Builder.php

```diff
@@ -47,7 +48,7 @@ class Shopware_Plugins_Frontend_NostoTagging_Components_User_Builder
     public function build($identity)
     {
         $metaData = new User();
-        if (!is_null($identity)) {
+        if ($identity !== null) {
```

Comment:
`$identity` is not type hinted. I would check that the `$identity->name` attribute exists